src/pages/Races.tsx

```diff
@@ -5,21 +5,39 @@ import { useAccount } from "wagmi";
 import { useWalletModal } from "../providers/WalletModal";
 import moment from "moment";
 import { RaceTable } from "../components/Races";
-import { PlaceBetModal } from "../components/Modals";
+import { PlaceBetModal, SettleBetModal } from "../components/Modals";
 import utils from "../utils";
 import { Runner } from "../types/meets";
 import { PageLayout } from "../components";
+import { useSubgraphBets } from "../hooks/subgraph";
+import { BetHistory } from "../types/bets";
+import { BetTable } from "../components/Bets";
+import { makeMarketId } from "../utils/markets";
+import { formatBytes16String } from "../utils/formatting";
+import { useConfig } from "../providers/Config";
 
 export const Races: React.FC = () => {
   const params = useParams();
   const track = params.track || "";
   const raceNumber = Number(params.number) || 0;
 
   const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isSettleModalOpen, setSettleIsModalOpen] = useState(false);
   const [selectedRunner, setSelectedRunner] = useState<Runner>();
-
+  const [selectedBet, setSelectedBet] = useState<BetHistory>();
   const { runners } = useRunnersData(track, raceNumber);
 
+  const config = useConfig();
+  const meetDate = moment().format("DD-MM-YY");
```

Comment:
maybe wrap in useMemo with empty dep array
might cause perf issue if there's a lot re-render